gstreamer: change priv_gst_parse_yylex arguments

Change priv_gst_parse_yylex to fit new bison version, else we will
get following error:

| grammar.tab.c: In function 'priv_gst_parse_yyparse':
| grammar.tab.c:67:25: error: too few arguments to function 'priv_gst_parse_yylex'
|  #define yylex           priv_gst_parse_yylex
|                          ^

Upstream-Status: Pending

Signed-off-by: Chong Lu <Chong.Lu@windriver.com>
---
 gst/parse/grammar.y | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/parse/grammar.y b/gst/parse/grammar.y
index 24fc87b..24fe906 100644
--- a/gst/parse/grammar.y
+++ b/gst/parse/grammar.y
@@ -36,7 +36,7 @@
 
 typedef void* yyscan_t;
 
-int priv_gst_parse_yylex (void * yylval_param , yyscan_t yyscanner);
+int priv_gst_parse_yylex (yyscan_t yyscanner);
 int priv_gst_parse_yylex_init (yyscan_t scanner);
 int priv_gst_parse_yylex_destroy (yyscan_t scanner);
 struct yy_buffer_state * priv_gst_parse_yy_scan_string (char* , yyscan_t);
-- 
1.9.1

